packages/twilio/demo/conversations/.env.example

```diff
@@ -0,0 +1,3 @@
+TWILIO_ACCOUNT_SID = xxxxxxxxxxxxxxx
```

Comment:
please move this outside of the library, instead an apps folder at the top level and move it there as twilio-conversations-demo